### PR TITLE
Add dynamic view render to Sidebar, hello world Reviews

### PR DESCRIPTION
**Pull request by augustdolan**

### CODE REVIEW

**Comment by augustdolan** [comment]
@hongmm When you're around, you mind setting some eyes on this? 

Thanks!

**Review by hongmm — APPROVED** [review]
@augustdolan these changes look good! I haven't played around with styled components so I am not 100% clear on exactly how it works but the way it's coded looks fine. Seems like the approach you are currently taking is to show the div when the state for showing the div is set to true which makes sense to me.

**Comment by augustdolan** [comment]
@Ahmed-Elawad  Could you re-review? I kept pushing here without realizing it. I just finished up my overview 

https://docs.google.com/document/d/1cvnYb2BXQvn-NfWw6sYo4POUBjd_65ptn7shr1nG208/edit

File: client/src/components/Reviews.jsx [hunk 1]
```diff
@@ -9,24 +10,90 @@ const FlexContainer = styled.div`
...
+            <ReviewBar rating={reviewsAverages.appearance} />
+            <Rating>{reviewsAverages.appearance}</Rating>
+          </Score>
+          <ScoreName>Works as expected</ScoreName>
+          <Score>
+            <ReviewBar rating={reviewsAverages.easeOfAssembly} />
+            <Rating>{reviewsAverages.easeOfAssembly}</Rating>
+          </Score>
+        </ScoreAverage>
+      </Averages>
```
Comment (Ahmed-Elawad) [inline client/src/components/Reviews.jsx:96]: @augustdolan, do you have a screenshot of this by chance?  Tough to visualize.

**Review by Ahmed-Elawad — APPROVED** [review]
@augustdolan, the changes here all look good. Just interested in seeing what your averages section is visually! Very compact in your code. Tough to visualize.

File: client/src/components/Reviews.jsx (hunk 1 above)
Comment (augustdolan) [reply client/src/components/Reviews.jsx:96]: Hey Ahmed! I think the link I provided should show you the overview!

Comment (Ahmed-Elawad) [reply client/src/components/Reviews.jsx:96]: Needed access to it, but. thank you for getting the image over to me! Looks great! 

image

**Pull request merged**